husky_components/search/main.js

```diff
@@ -22,8 +21,8 @@ define([], function() {
 
     var templates = {
             skeleton: [
-                '<a class="fa-search fa-flip-horizontal search-icon" href="#"></a>',
-                '<a class="fa-times-circle remove-icon" href="#"></a>',
+                '<span class="fa-search fa-flip-horizontal search-icon"></span>',
+                '<span class="fa-times-circle remove-icon"></span>',
```

Comment:
the problem is that you always have todo some weird stuff with `preventDefault` and `return false` which makes the event listener more complicated and hard to maintain. changing that to a `span` makes it much more useable. i have also added the css to show a pointer on this container.